sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3841,7 +3841,70 @@ pi_result piQueueFlush(pi_queue Queue) {
     // true).
     std::unique_lock<pi_shared_mutex> ContextsLock(
       Queue->Device->Platform->ContextsMutex, std::defer_lock);
-      
+
```

Comment:
Agree that we should add a comment note here that the following code is duplicated